src/screens/choice-exercises/ArticleChoiceExerciseScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
+import { DocumentType, ENDPOINTS } from '../../constants/endpoints'
+import axios from '../../utils/axios'
+import { RouteProp } from '@react-navigation/native'
+import { DocumentResultType, RoutesParamsType } from '../../navigation/NavigationTypes'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { Answer, ARTICLES } from '../../constants/data'
+import SingleChoiceExercise from './components/SingleChoiceExercise'
+
+interface LearnArticlesExerciseScreenPropsType {
+  route: RouteProp<RoutesParamsType, 'LearnArticles'>
+  navigation: StackNavigationProp<RoutesParamsType, 'LearnArticles'>
+}
+
+const ArticleChoiceExerciseScreen = ({ navigation, route }: LearnArticlesExerciseScreenPropsType) => {
+  const { extraParams } = route.params
+  const { trainingSetId } = extraParams
+  const [documents, setDocuments] = useState<DocumentType[]>([])
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+
+  const documentToAnswers = (document: DocumentType): Answer[] => {
+    return Object.values(ARTICLES).map(article => ({ article, word: document.word }))
+  }
+
+  useEffect(() => {
```

Comment:
Duplicated code here and in WordExerciseScreen. Think this is bad practice. Can you merge this into on Screen? 
You undid almost all of my changes for unifiying these two screens, didn't you?